Android/app/build.gradle

```diff
@@ -30,6 +30,7 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
```

Comment:
remove this file from commit

Reply:
Which file? Can you please explain it clearly?!

Reply:
app/build.gradle

Reply:
It's still here